Tasks/JenkinsQueueJob/jobqueue.ts

```diff
@@ -195,15 +195,15 @@ export class JobQueue {
             if (job.ExecutableNumber === -1) {
```

Comment:
Do you need to add a check for `ExecutableUrl` to this if statement? IOW is it guaranteed to be there if `ExecutableNumber` is there?

Reply:
It should be there, but even if it isn't it would simply render as empty space.

Reply:
Actually, the "replace" method would cause a problem if ExecutableUrl is undefined now. So, you should probably add a check for it. Perhaps something like "const url = ExecutableUrl && ExecutableUrl.replace..." just before jobContents.

Reply:
Good catch. I updated it as per your suggestion.